charts/trillian/templates/trillian-log-server/deployment.yaml

```diff
@@ -87,6 +88,11 @@ spec:
               value: {{ template "mysql.hostname" . }}
             - name: MYSQL_PORT
               value: {{ .Values.mysql.port | quote }}
+{{- else if .Values.logServer.storageSystemEnvCredentials }}
```

Comment:
is the intent to migrate the mysql usage to use `storageSystemEnvCredentials` as well?

Reply:
I hadn't thought about this. It could certainly be an option!